common/persistence/sql/sqlplugin/visibility.go

```diff
@@ -72,6 +73,11 @@ type (
 		ParentRunID          *string
 		RootWorkflowID       string
 		RootRunID            string
+
+		// Version must be at the end because the version column has to be the last column in the insert statement.
+		// Otherwise we may do partial updates as the version changes halfway through.
+		// This is because MySQL doesn't support row versioning in a way that prevents out-of-order updates.
+		Version int64 `db:"_version"`
```

Comment:
Please add a unit test to verify this is always the last field.

Reply:
Yep, Haven't resolved my reminder yet. Just fixing all the broken tests first.